acme/tests/messages_test.py

```diff
@@ -430,12 +421,15 @@ class CertificateRequestTest(unittest.TestCase):
     def setUp(self):
         from acme.messages import CertificateRequest
         self.req = CertificateRequest(csr=CSR)
+        # 'resource' gets stripped as a non-compliant field on serialization: restore it.
```

Comment:
Probably the same as the other question, but what change caused this?

Reply:
Yes, same explanation as above.